src/gtl/components/tile-view/tile-view.html

```diff
@@ -36,7 +36,7 @@
           </div>
           <div class="col-md-9 col-lg-9 col-xs-9 miq-info-section">
             <dl class="dl-horizontal tile">
-              <dt ng-repeat-start="(key, header) in config.columns | limitTo: 7 track by $index" ng-if="header.text && header.text.indexOf('Name') === -1" title="{{header.text}}">{{header.text}}:</dt>
+              <dt ng-repeat-start="(key, header) in config.columns | limitTo: 7 track by $index" ng-if="header.header_text && header.header_text.indexOf('Name') === -1" title="{{header.header_text}}">{{header.header_text}}:</dt>
               <dd ng-repeat-end ng-if="header.text && header.text.indexOf('Name') === -1" title="{{item.cells[key].text}}">{{item.cells[key].text | limitToSuffix : 25 : 25}}</dd>
```

Comment:
`header.text.indexOf('Name')` won't be true either, right?

Reply:
True.